### PR TITLE
readonly - added normalization for readonly modifier

### PR DESCRIPTION
**Pull request by mikedamay**

### CODE REVIEW

File: src/Exercism.Representers.CSharp/Normalization/RemoveReadOnly.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    public class RemoveReadOnly : CSharpSyntaxRewriter
+    {
+        public override SyntaxToken VisitToken(SyntaxToken token)
+        {
+            if (token.IsKind(SyntaxKind.ReadOnlyKeyword))
+                return default;
+
+            return base.VisitToken(token);
+        }
+
```
Comment (ErikSchierboom) [inline src/Exercism.Representers.CSharp/Normalization/RemoveReadOnly.cs:15]:
```suggestion
```

File: test/Exercism.Representers.CSharp.IntegrationTests/Exercism.Representers.CSharp.IntegrationTests.csproj [hunk 2]
```diff
@@ -19,6 +19,10 @@
     <SolutionFiles Include="Solutions\**" />
     <Compile Remove="@(SolutionFiles)" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="Solutions\ReadOnly\Fields\expected_representation.txt" />
+  </ItemGroup>
```
Comment (ErikSchierboom) [inline test/Exercism.Representers.CSharp.IntegrationTests/Exercism.Representers.CSharp.IntegrationTests.csproj:25]:
```suggestion
```

File: test/Exercism.Representers.CSharp.IntegrationTests/Solutions/ReadOnly/Fields/Fake.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+using System;
+
+public class Fake
+{
+    private readonly int roField = 1729;
```
Comment (ErikSchierboom) [inline test/Exercism.Representers.CSharp.IntegrationTests/Solutions/ReadOnly/Fields/Fake.cs:5]: Maybe also add a static field? It should not matter, but just for the test coverage.

**Review by ErikSchierboom — APPROVED** [review]
Excellent!

File: test/Exercism.Representers.CSharp.IntegrationTests/Solutions/ReadOnly/Fields/Fake.cs (hunk 3 above)
Comment (mikedamay) [reply test/Exercism.Representers.CSharp.IntegrationTests/Solutions/ReadOnly/Fields/Fake.cs:7]: added

**Comment by mikedamay** [comment]
@ErikSchierboom I don't have permissions to merge.

**Comment by ErikSchierboom** [comment]
@mikedamay One I just thought of one other thing. Could you maybe also add tests for the other cases where `readonly` can be used? Out of the top of my head, there are three other scenarios:

1. `readonly struct` definitions
2. `readonly` return types (including `ref readonly` and `public readonly` and `readonly get` property accessors)
3. `ref readonly` variables

**Comment by ErikSchierboom** [comment]
@mikedamay I've merged this PR, thanks for working on this! The tests I suggested adding in the above comment can be added it in follow-up PR.

**Pull request merged**